components/collector/tests/source_collectors/quality_time/test_source_up_to_dateness.py

```diff
@@ -18,7 +18,7 @@ class QualityTimeSourceUpToDatenessTest(QualityTimeTestCase):
     def setUp(self):
         """Set up test data."""
         super().setUp()
-        self.api_url = f"{self.url}/api/v3/reports"
+        self.api_url = f"{self.url}/api/internal/reports"
```

Comment:
I think the Quality-time collector should use the public API. In theory the source Quality-time could be a different version so it's better to go via a versioned endpoint.